src/main/java/com/auth0/json/mgmt/guardian/Enrollment.java

```diff
@@ -67,7 +67,7 @@ public String getType() {
     }
 
     /**
-     * Getter for the enrollment name. This is usually the phone number.
+     * Getter for the enrollment name. This is usually the phone name. Only for 'guardian' enrollments.
```

Comment:
API says this is "the enrollment name (usually the phone number)". Was the _name_ change also intended?